simulators/ethereum/engine/suites/withdrawals/README.md

```diff
@@ -6,8 +6,16 @@ This test suite verifies behavior of the Engine API on each client after the Sha
 ## Shanghai Fork
 - Genesis Shanghai: Tests the withdrawals fork happening since genesis (e.g. on a testnet).
 - Shanghai Fork on Block 1: Tests the withdrawals fork happening directly after genesis.
-- Shanghai Fork on Block 2: Tests the transition to the withdrawals fork after a single block has happened. Block 1 is sent with invalid non-null withdrawals payload and client is expected to respond with the appropriate error.
-- Shanghai Fork on Block 3: Tests the transition to the withdrawals fork after two blocks have happened. Block 2 is sent with invalid non-null withdrawals payload (both in `engine_newPayloadV2` and the attributes of `engine_forkchoiceUpdatedV2`) and client is expected to respond with the appropriate error.
+- Shanghai Fork on Block 2: Tests the transition to the withdrawals fork after a single block has happened. Block 1 is used to send invalid non-null withdrawals payload.
+- Shanghai Fork on Block 3: Tests the transition to the withdrawals fork after two blocks have happened. Block 1 and 2 are used to send invalid non-null withdrawals payload.
+
+All test cases contain following verifications:
+- Verify client responds with error -32602 on the following scenarios:
+  - Send `ExecutionPayloadV2` using a custom valid (correct BlockHash) execution payload that includes an empty array for withdrawals on `timestamp < SHANGHAI_TIMESTAMP` using `engine_newPayloadV2`
+  - Send `PayloadAttributesV2` using an empty array for withdrawals on `timestamp < SHANGHAI_TIMESTAMP` using `engine_forkchoiceUpdatedV2`
+  - Send `ExecutionPayloadV1` using a custom valid (correct BlockHash) execution payload that includes `null` as withdrawals on `timestamp >= SHANGHAI_TIMESTAMP` using `engine_newPayloadV2`
+  - Send `PayloadAttributesV2` using `null` for withdrawals on `timestamp < SHANGHAI_TIMESTAMP` using `engine_forkchoiceUpdatedV2`
+- Use `engine_forkchoiceUpdatedV2` and `engine_newPayloadV2` to send pre-Shanghai payloads/payload attributes, and verify method call succeeds.
```

Comment:
It's worth checking that `engine_getPayloadV2` does also return pre-Shanghai payload if pre-Shanghai attributes have been sent to the build process

Reply:
Added in 835db11807e12043e44a34076e73e99728bc03e3